test/mke2e/suite_test.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mke2e
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"sigs.k8s.io/kueue/test/util"
+)
+
+var (
+	k8sManagerClient client.Client
+	k8sWorker1Client client.Client
+	k8sWorker2Client client.Client
+	ctx              context.Context
+)
+
+const (
+	Timeout  = time.Minute
+	Interval = time.Millisecond * 250
+)
```

Comment:
Can we commonize these variables in 

https://github.com/kubernetes-sigs/kueue/blob/74377c7eb3e37db4703538abbfb9432718be4097/test/util/constants.go
?

Reply:
removed